README.md

```diff
@@ -67,6 +68,11 @@ Provides commands and jobs for running linters on projects. For information on c
 
 Provides npm utility functions. For information on available functions and configuration parameters, refer to the [documentation](https://circleci.com/orbs/registry/orb/arrai/npm).
 
+### pip-audit
+[![pip-audit: version][]](https://circleci.com/orbs/registry/orb/arrai/pip-audit)
```

Comment:
I thought prettier would want a newline here, like the rest?

Reply:
It would, but when I re-set up that repo on my machine I forgot to set up husky/lint-staged.